server.lua

```diff
@@ -6,38 +6,36 @@ RegisterNetEvent('qb-vineyard:server:getGrapes', function()
     Player.Functions.AddItem("grape", amount)
 end)
 
-QBCore.Functions.CreateCallback('qb-vineyard:server:loadIngredients', function(source, cb)
+lib.callback.register('qb-vineyard:server:loadIngredients', function(source)
 	local src = source
 	local Player = QBCore.Functions.GetPlayer(src)
-    local grape = Player.Functions.GetItemByName('grapejuice')
+    local grape = exports.ox_inventory:GetItem(src, 'grapejuice', nil, true)
 	if not Player.PlayerData.items then
```

Comment:
improper indent

Reply:
Very weird, this does not show on my editor.